authors/apps/articles/tests/test_articles.py

```diff
@@ -0,0 +1,208 @@
+from rest_framework.test import APIClient, APITestCase
+from django.urls import reverse
+from django.template.defaultfilters import slugify
+
+from ..models import Article
+from ...authentication.models import User
+
+
+class ArticleTestCase(APITestCase):
+    """
+        Article endpoints test
+    """
+
+    def setUp(self):
+        """
+            Setup method
+        """
+        # All articles urls
+        self.all_article_url = reverse("articles:articles")
+        self.login_url = reverse("auth:login")
+        self.register = reverse("auth:register")
+        self.client = APIClient()
+        self.valid_article_data = {
+            "article": {
+                "image_path": "......",
+                "title": "Its a test article",
+                "body": "Its a test article body"
+            }
+        }
+
+        self.invalid_article_data = {
+            "image_path": "......",
+            "title": "Its a test article",
+            "body": ""
+        }
+
+        self.update_article_data = {
+            "image_path": "......",
+            "title": "Its a test article",
+            "body": "ajdlkjasl;dkfjalk;sdjf"
+        }
+
+        self.user_data = {
+            "user": {
+                "username": "test",
+                "email": "test@fmail.com",
+                "password": "Test@254"
+            }
+        }
+
+    def login(self, data):
+        """
+            Login a registered user
+        """
+        # Register a user
+        self.client.post(
+            self.register,
+            data=data,
+            format="json"
+        )
+        response = self.client.post(
+            self.login_url,
+            data=data,
+            format="json"
+        )
+        return response.data['token']
+
+    def get_slug_from_title(self, title):
+        """
+            Get slug
+        """
+        specific_article_url = reverse(
+            "articles:specific_article",
+            kwargs={
+                "slug": slugify(title)
+            }
+        )
+        return specific_article_url
+
+    def create_article(self, data):
+        """
+            Create an article
+        """
+        article = Article(
+            image_path=data['image_path'],
+            slug=slugify(data['title']),
+            title=data['title'],
+            body=data['body'],
+            author_id=1
+        )
+        article.save()
+
+    def test_get_all_artcles(self):
+        """
+            Test GET /api/v1/articles/
+        """
+        response = self.client.get(
+            '/api/v1/articles/',
+            format="json"
+        )
+        self.assertEqual(response.status_code, 200)
+        data = response.data
+        self.assertTrue(len(data) == 0)
+
+    def test_get_one_article(self):
```

Comment:
Nice work on the test and the helper functions but also, test for the response data as well